### PR TITLE
Fix tasks tags in kfp workflows.

### PR DESCRIPTION
**Pull request by revit13**
## Why are these changes needed?

Currently the task tags in kfp workflow are not updated correctly. This PR  tries to fix that

### CODE REVIEW

File: .make.versions [hunk 1]
```diff
@@ -13,29 +13,45 @@ RELEASE_VERSION_SUFFIX=.dev6
 # Data prep lab wheel version
 DPK_LIB_VERSION=0.2.0$(RELEASE_VERSION_SUFFIX)
 DPK_LIB_KFP_VERSION=0.2.0$(RELEASE_VERSION_SUFFIX)
-DPK_LIB_KFP_VERSION_v2=0.2.0$(RELEASE_VERSION_SUFFIX)
```
Comment (roytman) [inline .make.versions]: I think we need a separate image for v2

**Review by roytman — APPROVED** [review]
LGTM, but would like that @daw3rd or @blublinsky will take a look.

File: .make.versions (hunk 1 above)
Comment (revit13) [reply .make.versions]: Fixed. Thanks

**Comment by daw3rd** [comment]
LGTM

**Pull request merged**